### PR TITLE
Change default location for storage.conf to...

### PR DESCRIPTION
**Pull request by dfr**
... /usr/local/etc/containers on FreeBSD

Signed-off-by: Doug Rabson <dfr@rabson.org>

### CODE REVIEW

**Comment by dfr** [comment]
Looks like I broke the linux build - I'll update the PR shortly

File: types/options_freebsd.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const (
+	// these are default path for run and graph root for rootful users
+	// for rootless path is constructed via getRootlessStorageOpts
+	defaultRunRoot   string = "/var/run/containers/storage"
+	defaultGraphRoot string = "/var/db/containers/storage"
+)
+
+// defaultConfigFile path to the system wide storage.conf file
+var (
+	defaultConfigFile         = "/usr/local/share/containers/storage.conf"
```
Comment (TomSweeneyRedHat) [inline types/options_freebsd.go:12]: should this be `/usr/local/containers/storage.conf` to match https://github.com/containers/storage/pull/1221/files#diff-3e70e441de9baa998963aaec77ca4cfe228f061b38d08eda62f28bdc53e3fdcbR8

Comment (dfr) [reply types/options_freebsd.go:12]: I think the example conf file should change instead. Generally, this kind of thing tends to live either in /usr/local/share or /usr/local/etc with any file likely to be changed locally strongly preferred to be in /usr/local/etc.

Comment (dfr) [reply types/options_freebsd.go:12]: I changed storage.conf-freebsd to match the compiled in default

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@giuseppe PTAL

**Comment by giuseppe** [comment]
LGTM

**Review by flouthoc — APPROVED** [review]
LGTM

File: types/options_freebsd.go (hunk 1 above)
Comment (TomSweeneyRedHat) [reply types/options_freebsd.go:12]: THat works for me, I was mostly concerned about the difference between the two.  Thanks @dfr

**Pull request merged**